clients/params.go

```diff
@@ -18,3 +20,21 @@ type OrgBucketParams struct {
 	OrgParams
 	BucketParams
 }
+
+type AuthLookupParams struct {
```

Comment:
I'd only expect this to be used across the v1_auth subcommands, so I'd suggest keeping it in `cmd/influx/v1_auth.go`